### PR TITLE
Go back to using threads to spawn images, removing async

### PR DESCRIPTION
**Pull request by trimental**
Switch back to threads due to performance challenges with async. Hopefully this should still perform better then before due to queuing up reposition events when image load in, as well as improving removing the logic to manually add a callback to the image loading thread.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <h1 align="center">Inlyne - a GPU powered, browserless, markdown + html viewer</h1>
 
 <p align="center">
-<a href="https://crates.io/crates/inlyne" target="_blank"><img height=25 src="https://img.shields.io/crates/v/inlyne.svg" /></a>
```
Comment (CosmicHorrorDev) [inline README.md:4]: Was this supposed to be removed?

File: README.md [hunk 2]
```diff
@@ -45,7 +44,6 @@ the document where you left off. It's designed to work seamlessly and allow you
 | Favourite band    | Nickleback         |
 
 #### Sizable images
-<img src="https://i.redd.it/hwurhp7crzf81.png" width = "300" align="center"></img>
```
Comment (CosmicHorrorDev) [inline README.md:48]: Ditto with above

**Review by CosmicHorrorDev — COMMENTED** [review]
Everything looks great to me! Just a couple of small comments :+1:

File: README.md (hunk 1 above)
Comment (trimental) [reply README.md:4]: Yes certainly not suppose to be removed oops. Was just testing if the presence of images induced any blocking. I'll fix this up thanks.

**Pull request merged**